data/mt/retired/Andrea-Olsen-2dcaf4be-0227-4ced-b01c-aacd8376c45a.yml

```diff
@@ -3,18 +3,24 @@ name: Andrea Olsen
 given_name: Andrea
 family_name: Olsen
 gender: Female
-email: Andrea.Olsen@MTLeg.gov
+email: Andrea.Olsen@legmt.gov
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-31'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '100'
+- end_date: '2022-12-31'
+  type: upper
```

Comment:
@bfossen-ce this person is listed in the retirements but they are actually starting a term in the senate district 50 on 01/02/2023, so this should be a start date I believe?